k8s/openebs-monitoring-pg.yaml

```diff
@@ -52,11 +52,9 @@ data:
       - source_labels: [__meta_kubernetes_pod_name]
         action: replace
         target_label: kubernetes_pod_name
-      - source_labels: [__meta_kubernetes_pod_label_vsm]
```

Comment:
cool! didn't know about this special char replace with _. 

On the change itself:
- should we leave it as it is for backward compatibility, possibly for another release till all the 0.6 volumes get upgraded? 
- the below openebs_pv can be related with this openebs.io/persistent-volume?

Reply:
Yes @kmova , found that special char can be replaced with _ in the link:
https://github.com/prometheus/prometheus/blob/release-2.3/documentation/examples/prometheus-kubernetes.yml.

Reply:
We can leave the vsm tag for backward compatibility sake. I'll raise another commit for the same.